paragon/ui/controllers/sprites.py

```diff
@@ -0,0 +1,542 @@
+from PySide2 import QtCore
```

Comment:
Since this file's pretty large, it might be better to break it. I think for cases like this lots of little files are easy to deal with than one big file.